proc_IR_noecho.py

```diff
@@ -35,12 +35,8 @@
             postproc=postproc, lookup=postproc_dict, 
             dimname='indirect')
     s.ift('t2')
-    rough_center = abs(s).convolve('t2',0.01).mean_all_but('t2').argmax('t2').item()
-    s.setaxis(t2-rough_center)
-    fl.next('rough centered')
-    fl.image(s)
-    #{{{convolution and zeroth order phase correction
-    s = s['t2':(0,None)]['ph2',0]['ph1',-1]
```

Comment:
there is no echo, so trying to center the data based on the maximum  doesn't make sense.  Also, slicing out the data starting from that point likewise doesn't make sense.

Is it possible that invrec_deutanal was actually the better version of this code.

If there is no echo, we really want to be running some type of baseline correction.

In particular, I want to know how to reproduce the baseline-corrected peaks from your 2nd year.

Reply:
So pertaining to #24, This code I am starting to work with and is called proc_90_pulse.py. for some reason it is going to the wrong postproc_dict. Will come back to this issue. But I will try to incorporate a baseline into proc_IR_noecho.py

Reply:
Are you sure you want to start from this? Where is the code that we're talking about [here](https://jmfrancklab.slack.com/archives/CV553TPRV/p1585181842001300) and [here](https://jmfrancklab.slack.com/archives/CKVFSAXST/p1585181683013100?thread_ts=1585158085.003500) and [here](https://jmfrancklab.slack.com/archives/CKVFSAXST/p1585181505012500?thread_ts=1585158085.003500)

Reply:
Those were code to process a simple 90 pulse on d2o and are in the processing script I call proc_90_pulse.py it is literally what we used to process the 2 years data but renamed. This is different that a simple 90 pulse

Reply:
OK, yes, I think we're on the same page now -- so posts above are using the code referenced in #24 ?

Reply:
yes I believe so. This script in particular, is an integration of both the manual phasing of a 90 pulse we worked on prior to two years, and invrec_deutanal. I am definitely positive we do no want invrec_deutanal as that had us manually doing the cost funciton (going through each individual indirect and finding the minimum and plugging that in for the next)